PublicMapping/districtbuilder-classic
# Incorporate bug fixes and improvements from DTL

## Overview

Incorporate bug fixes and some improvements (small features that would generally be useful, formatting, etc).

### Checklist


### Notes

I think the best way to review this is by reviewing the changes as a whole and then maybe going commit by commit. The changes should all be pretty straightforward.

Bug fixes:
* Make hosts configurable (https://github.com/azavea/district-builder-dtl-pa/pull/16)
* Fix translations (https://github.com/azavea/district-builder-dtl-pa/pull/32 and https://github.com/azavea/district-builder-dtl-pa/pull/40 and https://github.com/azavea/district-builder-dtl-pa/pull/60)
* Fix the way files are copied into the container (https://github.com/azavea/district-builder-dtl-pa/pull/43)
* Increase nginx buffer sizes (https://github.com/azavea/district-builder-dtl-pa/pull/109)
* Avoid Travis build timeouts (https://github.com/azavea/district-builder-dtl-pa/pull/93)
* Pin to specific version of kombu (https://github.com/azavea/district-builder-dtl-pa/pull/28)
* Avoid app deployment failures (https://github.com/azavea/district-builder-dtl-pa/pull/77)
* Fix timestamp bugs (https://github.com/azavea/district-builder-dtl-pa/pull/68)

Misc small improvements:
* Ignore more files
* Formatting
* Tiny UI improvements (white halo to labels, sort score display alphabetically)
* Support for new basemaps
* Change to allow setup script to update fields (https://github.com/azavea/district-builder-dtl-pa/pull/108)

## Testing Instructions

Either copy over `.env.sample` and fill in passwords again or make sure to add `HOST=localhost` to your `.env`.

Run app via `./scripts/update && ./scripts/server` and make sure translations work.

Closes #162251813
